pkg/scalers/redis_scaler.go

```diff
@@ -197,9 +197,30 @@ func getRedisListLength(ctx context.Context, address string, password string, li
 	}
 
 	client := redis.NewClient(options)
+	var listType *redis.StatusCmd
 
-	cmd := client.LLen(listName)
+	listType = client.Type(listName)
```

Comment:
Not sure if list name is appropriate here

Reply:
any other suggestions ?